settings.gradle.kts

```diff
@@ -28,3 +28,7 @@ buildCache {
         isPush = isMasterBranch
     }
 }
+
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version ("0.8.0")
```

Comment:
From the 8.0 release [notes](https://docs.gradle.org/8.0/release-notes.html):

> Gradle 7.6 introduced [toolchain repositories](https://docs.gradle.org/8.0/userguide/toolchains.html#sub:download_repositories) for increased flexibility. In Gradle 8.0, there is no longer a default toolchain provisioner. You have to declare at least one Java Toolchain repository explicitly. This can be done via toolchain repository plugins, like the [Foojay Toolchains Plugin](https://plugins.gradle.org/plugin/org.gradle.toolchains.foojay-resolver-convention)

Reply:
It is used to download JVMs basically